includes/Modules/AdSense.php

```diff
@@ -641,8 +641,7 @@ function( Google_Service_AdSense_Alert $alert ) {
 						$option                  = (array) $this->options->get( self::OPTION );
 						$option['setupComplete'] = true;
 						$option['clientID']      = $data['clientID'];
-						$option['useSnippet']    = ! empty( $data['useSnippet'] );
-
+						$option['useSnippet']    = empty( $data['useSnippet'] ) ? true : $data['useSnippet'];
```

Comment:
The logic here will always set `useSnippet` to `true` since there is no differentiation between whether or not `$data['useSnippet']` was passed or not or if it was passed as `false`.

Also, this should only set the default in the case that `$option['useSnippet']` is not set at all since the default value is already applied in the `option_` filter. This filter doesn't run when the option isn't saved yet so we only need to account for that here.

Reply:
Updated in https://github.com/google/site-kit-wp/pull/957/commits/04af618b76fdfa1f5ddcdd2d3cc0227d1fc1bbee